lpa-submissions-e2e-tests/cypress/support/common/completeQuestionnaire.js

```diff
@@ -1,4 +1,56 @@
-module.exports = () => {
+import { input, textBox, textArea } from '../PageObjects/common-page-objects';
+
+const uploadFiles = (fileName) => {
+  // start watching the POST requests
+  cy.server({ method: 'POST' });
+  cy.route({
+    method: 'POST',
+    url: 'upload',
+  }).as('upload');
+
+  cy.get('input#documents').attachFile([fileName]);
+
+  cy.wait('@upload', { requestTimeout: 3000 });
+  cy.server({ enable: false });
+};
+
+const stepCompletion = () => {
+  // Accuracy of submission
```

Comment:
we're going to need to move past the hard-coding + start parameterising this stuff pretty sharpish but i agree it solves todays problem 👍